static/src/js/components/ProjectPanels/ProjectPanels.js

```diff
@@ -332,10 +333,50 @@ class ProjectPanels extends PureComponent {
 
       const {
         title,
-        isCSDA: collectionIsCSDA
+        isCSDA: collectionIsCSDA,
+        cloudHosted,
+        duplicateCollections = []
       } = collectionMetadata
 
-      const { [collectionId]: collectionDataQualitySummaries = [] } = dataQualitySummaries
+      let { [collectionId]: collectionDataQualitySummaries = [] } = dataQualitySummaries
+
+      const hasDataQualitySummary = collectionDataQualitySummaries.length > 0
+      const hasDuplicateCollection = duplicateCollections.length > 0
+
+      const dataQualityHeader = (() => {
+        if (hasDataQualitySummary && hasDuplicateCollection) {
+          return 'Important data quality and availability information'
+        }
+        if (hasDataQualitySummary) {
+          return 'Important data quality information'
+        }
+        if (hasDuplicateCollection) {
+          return 'Important data availability information'
+        }
+        return ''
+      })()
+
+      if (hasDuplicateCollection) {
+        const duplicateCollectionId = duplicateCollections[0]
+        collectionDataQualitySummaries = [...collectionDataQualitySummaries, {
+          id: 'duplicate-collection',
+          summary: cloudHosted
+            ? (
+              <>
+                <span>This dataset is hosted in the cloud. The dataset is also </span>
```

Comment:
This should say "This dataset is hosted in the Earthdata Cloud..."

Reply:
done